Examples/Java/Sources/Board.java

```diff
@@ -0,0 +1,56 @@
+//
+// Board.java
+// Autogenerated by plank
+//
+// DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+// @generated
```

Comment:
getting `@Generated` on the type will be nice for static analysis tools to skip over analyzing it.
it's complicated to get the right `@Generated` annotation due to Java 9 changes.
no action to take right now. i'm investigating making a small library to make this easy.

Reply:
Sounds like a good follow-up issue once this lands